integration-tests/package.json

```diff
@@ -14,13 +14,13 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@frequency-chain/api-augment": "v0.9.29",
-        "@polkadot/api": "9.6.2",
-        "@polkadot/types": "^9.6.2",
-        "@polkadot/util": "^8.7.1",
+        "@polkadot/api": "9.10.2",
+        "@polkadot/types": "9.10.2",
+        "@polkadot/util": "10.2.1",
         "rxjs": "^7.5.7"
     },
     "devDependencies": {
-        "@polkadot/typegen": "^9.6.2",
+        "@polkadot/typegen": "9.10.2",
```

Comment:
Can we keep the version upgrade notation (^) so it is easier to get compatible updates?

Reply:
This contributed to the warnings so I think it's better to leave it without it for the time being.